data/hooks/conf_regen/12-metronome

```diff
@@ -24,9 +32,20 @@ do_pre_regen() {
   for domain in $YNH_DOMAINS; do
     cat domain.tpl.cfg.lua \
       | sed "s/{{ domain }}/${domain}/g" \
+      | sed "s/{{ HTTP_FILE_SECRET }}/${http_file_secret}/g" \
+      | sed "s/{{ HTTP_FILE_DELETE_SECRET }}/${http_file_delete_secret}/g" \
       > "${metronome_conf_dir}/${domain}.cfg.lua"
   done
 
+  # create the upload endpoint
+  [[ -d "/var/www/metronome/xmmp-upload" ]] \
+    || mkdir -p /var/www/metronome/upload/
```

Comment:
Seems a bug "xmpp-upload" vs "upload"

Reply:
aie ! yes it's a bug... will change all these soon

Reply:
fixed